kingpin/actors/aws/test/test_s3.py

```diff
@@ -30,10 +30,10 @@ def setUp(self):
                     'prefix': '/test',
                     'status': 'Enabled',
                     'expiration': '30',
-                    'transition': {
+                    'transitions': [{
```

Comment:
@LaikaN57 Does this mean you broke the API for people using `transition: ...`?

Reply:
Yup this broke with this change: https://github.com/Nextdoor/kingpin/pull/496/files#diff-8782cedb3640968fdf2505ccfc4f9d81R938